src/transformers/models/imagegpt/image_processing_imagegpt.py

```diff
@@ -199,6 +199,7 @@ def preprocess(
         do_normalize = do_normalize if do_normalize is not None else self.do_normalize
         do_color_quantize = do_color_quantize if do_color_quantize is not None else self.do_color_quantize
         clusters = clusters if clusters is not None else self.clusters
+        np.array(clusters)
```

Comment:
should assign it to `clusters`, otherwise no effect I believe.

Reply:
d'oh 🤦‍♀️